templates/boxMenuGroup.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { compile, classes } from 'core/js/reactHelpers';
+
+export default function BoxMenuGroup (props) {
+
+  const {
+    displayTitle,
+    body,
+    instruction
+  } = props;
+
+  return (
+    <div className="menu-group__inner boxmenu-group__inner">
+
+      <div className="menu-group__header boxmenu-group__header">
+        <div className="menu-group__header-inner boxmenu-group__header-inner">
+
+          {displayTitle &&
+          <div className="menu-group__title boxmenu-group__title">
+            <div className={classes([
```

Comment:
Perhaps splitting the attributes across multiple lines helps with readability? 
```suggestion
            <div 
              className={classes([
```